src/sentry/receivers/sentry_apps.py

```diff
@@ -38,17 +45,41 @@ def send_issue_ignored_webhook(project, user, group_list, **kwargs):
         send_workflow_webhooks(project.organization, issue, user, "issue.ignored")
 
 
+@comment_created.connect(weak=False)
+def send_comment_created_webhook(project, user, group, data, **kwargs):
+    send_workflow_webhooks(project.organization, group, user, "comment.created", data=data)
+
+
+@comment_updated.connect(weak=False)
+def send_comment_updated_webhook(project, user, group, data, **kwargs):
+    send_workflow_webhooks(project.organization, group, user, "comment.updated", data=data)
+
+
+@comment_deleted.connect(weak=False)
+def send_comment_deleted_webhook(project, user, group, data, **kwargs):
+    send_workflow_webhooks(project.organization, group, user, "comment.deleted", data=data)
```

Comment:
maybe this should be a separate function `send_comment_webhooks` that only fires the `build_comment_webhook` task

Reply:
Agh I went back and forth on that, cause technically comments _are_ a workflow event. Do you think it'd be clearer if I did that?

Reply:
yea cause then we don't need the ternary logic in `send_workflow_webhooks`